website/docs/components/rich-tooltip/partials/code/component-api.md

```diff
@@ -0,0 +1,125 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="[RT].Toggle" @type="yielded component">
+    The `RichTooltip::Toggle` component (see below).
+  </C.Property>
+  <C.Property @name="[RT].Bubble" @type="yielded component">
+    The `RichTooltip::Bubble` component (see below).
+  </C.Property>
+  <C.Property @name="isOpen" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Controls if the tooltip should be rendered initially opened.
+    <br />
+    <br />
+    _Notice: in this case the tooltip can't be dismissed via `esc` or "click outside" until the end user has interacted with it (it's in a "manual" state)._
+  </C.Property>
+  <C.Property @name="enableSoftEvents" @type="boolean" @default="true" @values={{array "true" "false"}}>
+    Assigns "soft" event listeners (`mouseEnter/Leave` + `focusIn/Out`) to the toggle to control the visibility of the tooltip.
+  </C.Property>
+  <C.Property @name="enableClickEvents" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Assigns a "click" event listener (`onClick`) to the toggle to control the visibility of the tooltip.
+    <br />
+    <br />
+    _Notice: enabling "click" event listeners will automatically disable "soft" event listeners._
+  </C.Property>
+  <C.Property @name="onOpen" @type="function">
+    A callback function invoked when the tooltip is opened (if provided).
+  </C.Property>
+  <C.Property @name="onClose" @type="function">
+    A callback function invoked when the tooltip is closed (if provided).
+  </C.Property>
+  <C.Property @name="[RT].isOpen" @type="yielded tracked property">
+    Hook into this tracked property to access the state of `isOpen`.
+  </C.Property>
+  <C.Property @name="[RT].close" @type="yielded function">
+    Hook into this function to programmatically close the toggle.
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+### Contextual components
+
+The Toggle and Bubble elements are passed into the Rich Tooltip as yielded components, using the `Toggle` and `Bubble` keys.
+
+#### RichTooltip::Toggle
+
+Standard toggle element to use, made of a text and an optional icon, to ensure that the toggle is perceivable, visually consistent, and can be used inline with other content or standalone as part of the layout flow.
+
+It can also be used with generic content, in which case consumers will need to make sure the component is used in a [conformant accessible way](/components/rich-tooltip?tab=accessibility).
```

Comment:
```suggestion
It can also be used with generic content, in which case consumers will need to ensure the component is used in a [conformant accessible way](/components/rich-tooltip?tab=accessibility).
```